### PR TITLE
Coloured events

### PR DESCRIPTION
**Pull request by nyb12**

### CODE REVIEW

File: apps/front-end/src/pages/front-end/orientation/Orientation.js [hunk 1]
```diff
@@ -396,20 +396,22 @@ export default function Orientation({ footerLinks }) {
               <VStack space="4" mt="4">
                 <HStack alignItems="Center" space="md">
                   <CheckCircleIcon size="4" color="blue.500" />
-                  <AdminTypo.H6 bold>{t("INTERVIEW")}</AdminTypo.H6>
+                  <AdminTypo.H6 bold>Prerak Orientation</AdminTypo.H6>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/orientation/Orientation.js:399]: Get this from enum API and show accordingly.

File: apps/front-end/src/pages/front-end/orientation/Orientation.js [hunk 2]
```diff
@@ -467,6 +469,12 @@ export default function Orientation({ footerLinks }) {
                       ? item?.location_type
                       : "",
                     event_id: item?.id ? item?.id : "",
+                    backgroundColor:
+                      item?.type === "Prerak Orientation"
+                        ? "#3B82F6"
+                        : item?.type === "Prerak FLN Training"
+                        ? "#5DC65F"
+                        : "#EAB431",
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/orientation/Orientation.js:477]: Use enum API color

Comment (nyb12) [reply apps/front-end/src/pages/front-end/orientation/Orientation.js:477]: added in constants in common-lib

File: apps/front-end/src/pages/front-end/orientation/Orientation.js [hunk 3]
```diff
@@ -467,6 +483,12 @@ export default function Orientation({ footerLinks }) {
                       ? item?.location_type
                       : "",
                     event_id: item?.id ? item?.id : "",
+                    backgroundColor:
+                      item?.type === "prerak_orientation"
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/orientation/Orientation.js:487]: change to `EVENTS_COLORS[item?.type]`

File: lib/common-lib/src/config/constant.js [hunk 4]
```diff
@@ -1 +1,7 @@
 export const REACT_APP_UPLOAD_FILE_SIZE = 10
+
+export const EVENTS_COLORS = {
+  blue: '#3B82F6',
+  green: '#5DC65F',
+  yellow: '#EAB431'
```
Comment (sagarkoshti1990) [inline lib/common-lib/src/config/constant.js:6]: change to event type value from enum API.

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**